bitsv/network/rates.py

```diff
@@ -98,23 +120,40 @@ def bsv_to_satoshi():
     return BSV
 
 
+def get_satoshi_to_fiat_rate(usd_per_bsv):
```

Comment:
We are pretty inconsistent with get_ use vs nothing vs set_. In this case I'd consider not having the function since it's only used once.